packages/react-vapor/src/components/textarea/TextArea.tsx

```diff
@@ -31,6 +28,10 @@ export interface ITextAreaOwnProps {
     isAutosize?: boolean;
 
     onChangeCallback?: (event: React.ChangeEvent<HTMLTextAreaElement>) => void;
+    validate?: (value: string) => boolean;
+    validationMessage?: string;
+    validationLabelProps?: ILabelProps;
+    children?: any;

```

Comment:
There is no need to declare children here, it is already implied when using a react component.